src/daemon/config.rs

```diff
@@ -922,6 +980,27 @@ mod tests {
         let c = Config::read_config("./defaults/krill.conf").unwrap();
         let expected_socket_addr: SocketAddr = ([127, 0, 0, 1], 3000).into();
         assert_eq!(c.socket_addr(), expected_socket_addr);
+        assert!(c.testbed().is_none());
+    }
+
+    #[test]
+    fn should_parse_testbed_config_file() {
+        // Config for auth token is required! If there is nothing in the conf
+        // file, then an environment variable must be set.
+        env::set_var(KRILL_ENV_AUTH_TOKEN, "secret");
+
+        let c = Config::read_config("./defaults/krill-testbed.conf").unwrap();
+        let expected_socket_addr: SocketAddr = ([127, 0, 0, 1], 3000).into();
```

Comment:
Why test the socket addr here? It's presumably tested elsewhere as it is part of the main Krill config and not part of the testbed config?

Reply:
quite right! removed this..